vendor/github.com/swiftstack/sortedmap/btree.go

```diff
@@ -83,12 +83,12 @@ type onDiskNodeStruct struct {
 	//                                  counted <N-1> number of Key:onDiskReferenceToNodeStruct pairs
 }
 
-type onDiskReferenceKeyStruct struct { // Used as Key for staleOndiskReferences LLRBTree below
+type onDiskReferenceKeyStruct struct { // Used as Key for staleOnDiskReferences LLRBTree below
```

Comment:
I'm just gonna put this comment here to remind us that these changes actually need to find there way into the corresponding file in the actual sortedmap repo. Here, you've changes the vendorized "copy" of the file that would, of course, be overwritten the next "glide up".

I'll take care of copying them over along with other Dispose() misses we talked about this morning I found inspired by you. Indeed, I now recall that I never really finished the Dispose() callback code - it came along back when we were looking at GC activities at many levels... and I believe we identified sortedmap as needing serious completion work that never happened. My oops!

Reply:
Apparently I haven't fixed all of the problems yet!  I just deleted my "huge" (2 Gibyte) file system and there are lots of __checkpoint__ objects still there.  Investigating ...

Reply:
I need to look at your other Dispose() fixes.  apparently i haven't fixed all the problems after all.